### PR TITLE
Deploy v1.0

### PR DESCRIPTION
**Pull request by cgarciaidentidad**
This is the first version of our software.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
```
Comment (camilogarcia-gmail) [inline index.html:4]: please, use a meta call viewport to be responsive

File: secondpage.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Second Page</title>
+</head>
+<body>
+	<ul>
+		<li>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iste, fuga porro voluptates autem animi deleniti iure maxime at architecto, minus aperiam aspernatur beatae voluptas nostrum. Cumque temporibus iusto voluptatum culpa.</li>
+		<li>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus aliquid, corrupti quidem dicta labore quod! Rem animi et laboriosam unde aspernatur neque labore dolorum quam maiores ipsam, facilis illum, facere.</li>
```
Comment (camilogarcia-gmail) [inline secondpage.html:10]: Here, I would like only 3 lines....

**Review by camilogarcia-gmail — CHANGES_REQUESTED** [review]
Please, take a lot of I am looking for

File: index.html (hunk 1 above)
Comment (cgarciaidentidad) [reply index.html:4]: No, no me da la gana....

File: secondpage.html (hunk 2 above)
Comment (cgarciaidentidad) [reply secondpage.html:10]: Aqui, eran 5 las lineas asi que suerte agonia....

File: index.html (hunk 1 above)
Comment (camilogarcia-gmail) [reply index.html:4]: Ok, lo hablaremos con el Jefe

File: secondpage.html (hunk 2 above)
Comment (camilogarcia-gmail) [reply secondpage.html:10]: Ok no habia necesidad de ser un HP

**Review by camilogarcia-gmail — APPROVED** [review]

**Pull request merged**